edxapp.sql

```diff
@@ -8817,6 +8817,7 @@ CREATE TABLE `oauth2_provider_application` (
 LOCK TABLES `oauth2_provider_application` WRITE;
 /*!40000 ALTER TABLE `oauth2_provider_application` DISABLE KEYS */;
 INSERT INTO `oauth2_provider_application` VALUES (1,'login-service-client-id','','public','password','X4L6BKKf0JgMn1TH0BFnmFLnKRplLYKmUYbqGc2qQLRIGzFjGiwQMPnIOyHMLiW6A8qsVX9mshchCUCcQGLELj5O4eZKCeeEacHON8JikewexicMCCw22gtxwkO4eGPu','Login Service for JWT Cookies',2,0,'2021-07-30 19:58:11.944124','2021-07-30 19:58:11.944166'),(2,'AjEA1P865dsBif42kQYMWJiLLGKQSVSYXy5X6o1V','','confidential','client-credentials','sZuhQuuV3QtpgpXqY6wXrPhB09R81Ijk8CBfzoSP9NER1xYEoLBaqKSggXzvCCjjIKOmytxV6Is3ClqqKMmRZtXTau7r9jrx5dfdgXRCGMSEIFy4xrWOC7Af3F8JP3jY','retirement',9,0,'2021-07-30 20:15:27.098232','2021-07-30 20:15:27.098284'),(3,'ecommerce-sso-key','http://localhost:18130/complete/edx-oauth2/ http://ecommerce.pearson.localhost:18130/complete/edx-oauth2/ http://ecommerce.main.localhost:18130/complete/edx-oauth2/','confidential','authorization-code','ecommerce-sso-secret','ecommerce-sso',1,1,'2021-07-30 20:18:16.419993','2021-07-30 20:18:16.420058'),(4,'ecommerce-backend-service-key','','confidential','client-credentials','ecommerce-backend-service-secret','ecommerce-backend-service',1,0,'2021-07-30 20:18:28.452549','2021-07-30 20:18:28.452606');
+UPDATE oauth2_provider_application SET redirect_uris='http://localhost:18381/complete/edx-oauth2/ http://discovery.pearson.localhost:18381/complete/edx-oauth2/ http://discovery.main.localhost:18381/complete/edx-oauth2/' WHERE id=8;
```

Comment:
What do you think about targeting the name instead of ID. In case an additional record is created before discovery-sso, then this ID might be different. @alexjmpb 
```suggestion
UPDATE oauth2_provider_application SET redirect_uris='http://localhost:18381/complete/edx-oauth2/ http://discovery.pearson.localhost:18381/complete/edx-oauth2/ http://discovery.main.localhost:18381/complete/edx-oauth2/' WHERE client_id='discovery-sso-key';
```

Reply:
I moved the command so it can be executed after the discovery provision, in the current form it seems it wasn't making any effect